plugins/GSdx/GSCrc.cpp

```diff
@@ -236,7 +241,7 @@ CRC::Game CRC::m_games[] =
 	{0x56242EC9, Genji, KO, 0},
 	{0xCDAF243D, Genji, CH, 0}, 
 	{0x2A5E0B61, Genji, CH, 0},
-	{0x7D4EA48F, Genji, NoRegion, 0},
+	// {0x7D4EA48F, Genji, EU, 0}, // same CRC as {0x7D4EA48F, HauntingGround, EU, 0},
```

Comment:
Don't change this. `0x7D4EA48F` was a duplicate for hunting ground and genji, commit 0dc1e7f4bba63f28bbb5cbf32bc7ea647f2ee97d commented the hunting ground one - leaving it only for genji, so why did you now comment this line - which removes `0x7D4EA48F` completely so it doesn't get used at all?